contracts/src/v0.8/dev/automation/2_1/AutomationForwarder.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import {AutomationRegistryBaseInterface as IRegistry} from "./interfaces/AutomationRegistryInterface2_1.sol";
```

Comment:
not related to this PR but I'm just curious, do we have separate AutomationRegistryBaseInterface and AutomationRegistryInterface ? Any reason to have both?